client/modules/IDE/components/Console.jsx

```diff
@@ -31,13 +36,15 @@ class Console extends React.Component {
       LOG_WARN_ICON: `url(${warnLightUrl})`,
       LOG_ERROR_ICON: `url(${errorLightUrl})`,
       LOG_DEBUG_ICON: `url(${debugLightUrl})`,
-      LOG_INFO_ICON: `url(${infoLightUrl})`
+      LOG_INFO_ICON: `url(${infoLightUrl})`,
+      BASE_FONT_SIZE: `${this.props.fontSize}`
```

Comment:
Can you move `BASE_FONT_SIZE` to another variable name? l think it's a little confusing to put it in `CONSOLE_FEED_LIGHT_ICONS`